### PR TITLE
live-common & libcore bump

### PR DESCRIPTION
**Pull request by gre**
like https://github.com/LedgerHQ/ledger-live-desktop/pull/2443

### CODE REVIEW

**Comment by gre** [comment]
it's pretty hilarious. it feels an issue in AsyncStorage. i'll take a look later. or someone else can.

We also need to switch to a better.

**Comment by gre** [comment]
bug should be fixed now. PR is ready.

File: src/screens/AddAccounts/02-SelectDevice.js [hunk 1]
```diff
@@ -39,6 +40,13 @@ class AddAccountsSelectDevice extends Component<Props, State> {
     ),
   };
 
+  comopnentDidMount() {
```
[the remark this answers is not in the document]
Comment (juan-cortes) [reply src/screens/AddAccounts/02-SelectDevice.js]: hawkeye

File: src/live-common-setup.js [hunk 2]
```diff
@@ -29,7 +29,7 @@ setSupportedCurrencies([
   "ethereum_classic",
   "ethereum",
   "hcash",
-  //  "tezos",
+  "tezos",
```
Comment (juan-cortes) [inline src/live-common-setup.js]: yes

File: src/screens/AddAccounts/02-SelectDevice.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (gre) [reply src/screens/AddAccounts/02-SelectDevice.js]: MortalSniper

**Pull request merged**